deploy/tests/test_yaml_api.py

```diff
@@ -169,14 +190,6 @@ def scy_runtime(tmp_path: Path, sample_backup_configs: Path) -> TmpYaml:
 def run_yamls(scy_hwdb: TmpYaml, scy_runtime: TmpYaml) -> RunTmpYamlSet:
     return RunTmpYamlSet(scy_hwdb, scy_runtime)
```

Comment:
Need to add the `non_existent_file` fixture to `run_yamls` fixture as well as `build_yamls`. 🤦‍♂️

Reply:
addressed in https://github.com/firesim/firesim/pull/1145/commits/eb55afb76e61a785dbff45e4712da1150f01d4c3